CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## Unreleased
+## [2.0.4]
```

Comment:
I would go for a minor version bump for a change like this as Hive is a big part of what WD is all about - i.e. 2.1.0.

Reply:
OK

Reply:
So are you going to bump it? Here and in the poms.

Reply:
yes